### PR TITLE
adds outfits for robot shipside roles

### PR DESCRIPTION
**Pull request by coldironwarrior**
## About The Pull Request

creates robot outfits for: chief medical officer, medical officer, medical researcher; staff officer; captain; and associated rebel roles

robots will now spawn with all their role-related pocket and belt goodies, as well as a default robotic armor mount

captain and staff officer robot outfits include light body armor and a helmet.

## Why It's Good For The Game

important and limited role-related gear should be available to players regardless of race they play

## Changelog

:cl:
fix: Combat robots spawning in roles that have roundstart gear will now have that gear equipped, along with a robotic armor mount
/:cl:

### CODE REVIEW

**Comment by Gregormack** [comment]
Any screenshots? no?

**Comment by Ragecrabbo** [comment]
As someone who made the mistake of picking CMO as my robot character and instantly necrotizing every person i attempted surgery on due to lacking medical clothes i absolutely welcome this PR.

**Comment by ClosetedSkeletonGH** [comment]
> As someone who made the mistake of picking CMO as robot character and instantly necrotizing every person i attempted surgery on due to lacking medical clothes i absolutely welcome this PR.

this doesn't fix this at all, make an issue report. This just makes CMO robots spawn with a belt

**Comment by ZAMODA** [comment]
May I make a request to allow robots to wear more decorative hats (cowboy hats, sunhats, etc) or would that be too out of scope for this PR?

**Comment by Lumipharon** [comment]
Why are smelly shipside roles spawning with combat armor?

**Comment by coldironwarrior** [comment]
> Why are smelly shipside roles spawning with combat armor?
 fixed

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -74,10 +79,28 @@ Godspeed, captain! And remember, you are not above the law."})
...
+	r_store = /obj/item/storage/pouch/general/large/command
+	l_store = /obj/item/hud_tablet/leadership
+
 /datum/outfit/job/command/captain/rebel
 	jobtype = /datum/job/terragov/command/captain/rebel
 	ears = /obj/item/radio/headset/mainship/mcom/rebel
 
+/datum/outfit/job/command/captain_robot/rebel
+	jobtype = /datum/job/terragov/command/captain/rebel
+	ears = /obj/item/radio/headset/mainship/mcom/rebel
```
Comment (Lumipharon) [inline code/datums/jobs/job/shipside.dm:103]: you need to add this outfit to the rebel job, same goes for the others.

**Review by Lumipharon — CHANGES_REQUESTED** [review]
Also I seem to have made the rebel FC have the rebel robo outfit, but not the rebel human outfit. If you could do me a favour and update that in this pr as well?

**Comment by QualityVan** [comment]
Still working on this? It'd be nice to get it in before/alongside #13190 since that'll prevent the latter from torturing robots.

**Comment by coldironwarrior** [comment]
i just deleted the rebel stuff since it got removed

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**